src/window.rs

```diff
@@ -530,50 +511,18 @@ impl WindowBuilder {
         mut self,
         parent_window: Option<rwh_06::RawWindowHandle>,
     ) -> Self {
-        self.window.parent_window = parent_window.map(SendSyncRawWindowHandle);
+        self.parent_window = parent_window.map(SendSyncRawWindowHandle);
         self
     }
-
-    /// Builds the window.
-    ///
-    /// Possible causes of error include denied permission, incompatible system, and lack of memory.
-    ///
-    /// ## Platform-specific
-    ///
-    /// - **Web:** The window is created but not inserted into the web page automatically. Please
-    ///   see the web platform module for more information.
-    #[inline]
-    pub fn build(self, window_target: &EventLoopWindowTarget) -> Result<Window, OsError> {
-        let window = platform_impl::Window::new(&window_target.p, self.window)?;
-        window.maybe_queue_on_main(|w| w.request_redraw());
-        Ok(Window { window })
-    }
 }
 
 /// Base Window functions.
 impl Window {
-    /// Creates a new Window for platforms where this is appropriate.
-    ///
-    /// This function is equivalent to [`Window::builder().build(event_loop)`].
-    ///
-    /// Error should be very rare and only occur in case of permission denied, incompatible system,
-    ///  out of memory, etc.
-    ///
-    /// ## Platform-specific
-    ///
-    /// - **Web:** The window is created but not inserted into the web page automatically. Please
-    ///   see the web platform module for more information.
-    ///
-    /// [`Window::builder().build(event_loop)`]: WindowBuilder::build
-    #[inline]
-    pub fn new(event_loop: &EventLoopWindowTarget) -> Result<Window, OsError> {
-        Window::builder().build(event_loop)
-    }
-
-    /// Create a new [`WindowBuilder`] which allows modifying the window's attributes before creation.
+    /// Create a new [`WindowAttributes`] which allows modifying the window's attributes before
+    /// creation.
     #[inline]
-    pub fn builder() -> WindowBuilder {
-        WindowBuilder::default()
+    pub fn attributes() -> WindowAttributes {
```

Comment:
I'd prefer this name as it more clearly says what it is.

```suggestion
    pub fn default_attributes() -> WindowAttributes {
```